tensorflow/core/framework/allocator.h

```diff
@@ -152,6 +152,7 @@ class Allocator {
   // allocated by this allocator.
   virtual size_t RequestedSize(void* ptr) {
     CHECK(false) << "allocator doesn't track sizes";
+    return size_t(0);
```

Comment:
It works in Release/RelWithDebInfo builds though, right?

I'd prefer to keep these as-is for now, as I think there are other issues that prevent a Debug build from succeeding (something about taking the address of zero-length iterators, if my memory serves me). Can you split these changes out and we'll consider them separately?